### ISSUE TITLE
Training won't start

### ISSUE THREAD

**Issue by Jofloku** [iperov/DeepFaceLab#534]
I have completed Steps 1 to 5 but now I want to start training, run training H64. But It Won't start training. Same Info appears at other trainingsetups. 
Running trainer. That's all that happened:

Loading model...

Model first run.
Enable autobackup? (y/n ?:help skip:n) : n
Write preview history? (y/n ?:help skip:n) : n
Target iteration (skip:unlimited/default) : default
0
Batch_size (?:help skip:0) : 20
Flip faces randomly? (y/n ?:help skip:y) : y
Use lightweight autoencoder? (y/n, ?:help skip:n) : n
Use pixel loss? (y/n, ?:help skip: n/default ) : default
Using plaidml.keras.backend backend.
INFO:plaidml:Opening device "opencl_amd_gfx900.0"
